schema/field/gen/numeric.tmpl

```diff
@@ -6,7 +6,7 @@
 package field
 
 import (
-	"errors"
+ 	"errors"
```

Comment:
Is it needed?

Reply:
No! Sorry, didn't catch this when reviewing. Updated now.